### DISCUSSION TITLE
Custom components as Accordion control/label

### DISCUSSION THREAD

**Discussion by naingthet** [mantinedev/mantine#5857] [Q&A]
Hi there, does anyone know if it's possible to use custom components as the label for accordion items? For example, using a `TextInput` or `Card` component as the Accordion label, and disabling the default click behavior. I would like to limit the open/close functionality to the chevron icon.

**Comment by rtivital** [comment]
It is not supported. If you do that, you will have accessibility violations.

**Reply by naingthet** [reply]
I see, thank you for the quick response!
